packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutput.tsx

```diff
@@ -154,6 +157,19 @@ export const TransactionReviewOutput = forwardRef<HTMLDivElement, TransactionRev
                     plainValue: true,
                 },
             ];
+        } else if (type === 'data' && replaceTxStakeType) {
```

Comment:
could you please merge 
`else if (type === 'data' && replaceTxStakeType)` and `} else if (type === 'address' && ethereumStakeType) {` conditions?